docs/sorting-algorithms/heap-sort.mdx

```diff
@@ -0,0 +1,326 @@
+---
+title: Heap Sort
+slug: /sorting-algorithms/heap-sort
+---
+
+# Introduction
+
+Heap Sort is a sorting algorithm which extracts the maximum or minimum element from the array and swap it with the last element of the array for sorting in ascending or descending order N times.
+It is similar to Selection Sort, the only difference is that it uses Heap data structure for finding minimum or maximum element.
+
```

Comment:
Can you add an explanation for the algorithm?

Reply:
done